editor-integration/emacs/dune.el

```diff
@@ -0,0 +1,54 @@
+;;; dune.el --- Align words in an intelligent way
+
+;; Copyright 2018 Jane Street Group, LLC <opensource@janestreet.com>
+;; URL: https://github.com/ocaml/dune
+;; Version: 1.0
+
+;;; Commentary:
+
+;; This package provides helper functions for interacting with the
+;; dune build system from emacs.
+
+;; Installation:
+;; You need to install the OCaml program ``dune''.  The
+;; easiest way to do so is to install the opam package manager:
+;;
+;;   https://opam.ocaml.org/doc/Install.html
+;;
+;; and then run "opam install dune".
+
+;;; Code:
+
+(defgroup dune nil
+  "Integration with the dune build system."
+  :tag "Dune build system."
+  :version "1.0"
+  :group 'align)
+
+(defcustom dune-command "dune"
+  "The dune command."
+  :type 'string
+  :group 'dune)
```

Comment:
the `defcustom`s following a `defgroup` don't have to specify the `:group` because the group is inferred

Reply:
Thanks, I didn't know that. Could you submit a PR to remove them?